test/parallel/test-fs-cp.mjs

```diff
@@ -810,6 +875,32 @@ if (!isWindows) {
   assertDirEquivalent(src, dest);
 }
 
+// It copies a nested folder structure with mode flags.
+// This test is based on fs.promises.copyFile() with `COPYFILE_FICLONE_FORCE`.
+await (async () => {
```

Comment:
```suggestion
{
```

Reply:
fixed in https://github.com/nodejs/node/pull/47084/commits/6a2b4bc7a566568ca65d5bcfc255a5761e424289